ftd2xx/aio/aio.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import ctypes as c
+from ..defines import *
+from ..ftd2xx import *
+from ..ftd2xx import _ft
+
+
+class FTD2XX(FTD2XX):
+    _timeouts = 0, 0
+    
+    @property
+    def timeouts(self): return self._timeouts
+    
+    @timeouts.setter
+    def timeouts(self, timeouts: tuple):
+        super().setTimeouts(*[1 if t > 0 else 0 for t in timeouts])
```

Comment:
I guess that implementing async write is necessary before merge...

Reply:
@snmishra do you have any suggestions on how to approach the write? The timeout is for data into FIFO or is it data transmitted?